src/components/users/Container.js

```diff
@@ -4,33 +4,62 @@ import { Route } from 'react-router-dom'
 import List from './List/List'
 import PostsContainer from '../posts/Container'
 
+import * as api from '../../api/users'
+
+
+// users: [
+//   {
+//     _id: '5de4',
+//     username: 'example.user',
+//     posts: [
+//       {
+//         _id: '6cj2',
+//         content: 'This is an example post.',
+//         emotion: 'joy',
+//         created_at: new Date('2019-07-01')
+//       }
+//     ]
+//   }
+// ]
 export default class Container extends React.Component {
   constructor (props) {
     super(props)
     this.state = {
-      users: [
-        {
-          _id: '5de4',
-          username: 'example.user',
-          posts: [
-            {
-              _id: '6cj2',
-              content: 'This is an example post.',
-              emotion: 'joy',
-              created_at: new Date('2019-07-01')
-            }
-          ]
-        }
-      ]
+      users: [],
+      loading: true
     }
+
+    this.refreshUsers = this.refreshUsers.bind(this)
+  }
+
+  async componentDidMount () {
+    this.refreshUsers().then(() => this.setState({ loading: false }))
+  }
+
+  async refreshUsers () {
+    const { response } = await api.getAllUsers()
```

Comment:
If you log what is being returned you will see it is the array, remove the brackets so there is no destructuring.